src/System.Private.ServiceModel/tests/Common/TestPropertiesTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.ServiceModel.Tests.Common;
+using Xunit;
+
+public static class TestPropertiesTest
+{
+    [Fact]
+    public static void TestProperties_Are_Initialized()
+    {
+        Assert.True(TestProperties.Instance.Properties.ContainsKey(TestProperties.BridgeResourceFolder_PropertyName),
```

Comment:
This unit test is necessary to demonstrate both that the code-gen happens and also to show what usage will look like by test code.  The actual values being tested does not have to be exhaustive.